src/pages/Portfolio/LendingStats/LendingStatsPage.vue

```diff
@@ -157,18 +157,15 @@ export default {
 	},
 	created() {
 		this.apollo.query({
-			query: userAchievementsProgress,
-			variables: {
-				userId: this.userId.toString(),
-			},
+			query: userAchievementProgressQuery,
 		}).then(({ data }) => {
-			this.allAchievements = data?.userAchievementProgress?.achievementProgress ?? [];
+			this.allAchievements = data?.userAchievementProgress?.lendingAchievements ?? [];
```

Comment:
The lender profile doesn't currently show tiered badges, correct? It looks like these changes utilize the old `lendingAchievements` query.

Reply:
Not sure what you mean. is this helpful?
https://kiva.atlassian.net/browse/MP-923?focusedCommentId=359210

Reply:
Does `lendingAchievements` include the new tiered badges as well? Or are those not included yet? And the comment mentions checking all of the milestones, but it doesn't look like we're doing that yet. And I don't know if this code covers the climate badge situation.

Reply:
Oh no, they are separated as you see in your comment about the query below.

Reply:
Okay makes sense. What about checking all milestones of an old achievement, like John mentioned? It looks like we're just checking the first milestone. And do we need special casing for the climate badge?

Reply:
Based on John's comment:

> for the lendingMilestones (the non-tiered ones) they are complete if all the milestones are complete

The first one will be complete if all are complete, for the edge case as climate badge.

Reply:
Ah gotcha. Do we have special casing already for the climate badge?

Reply:
Seems like not. Do you see smt different?

Reply:
Not seeing anything, but if we haven't had it, maybe we don't need the special casing here.